### PR TITLE
Add version rollback information

### PR DESCRIPTION
**Pull request by dadahelloworld**
## Proposed changes

1.Add version rollback information.
2.Fix typos.

## Types of changes

What types of changes does your code introduce to Doris?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update (if none of the other choices apply)
- [ ] Code refactor (Modify the code structure, format the code, etc...)
- [ ] Optimization. Including functional usability improvements and performance improvements.
- [ ] Dependency. Such as changes related to third-party components.
- [ ] Other.

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] I have created an issue on (Fix #ISSUE) and described the bug/feature there in detail
- [ ] Compiling and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If these changes need document changes, I have updated the document
- [x] Any dependent changes have been merged

## Further comments

If this is a relatively large or complex change, kick off the discussion at dev@doris.apache.org by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: docs/zh-CN/installing/upgrade.md [hunk 1]
```diff
@@ -80,3 +80,12 @@ Doris 可以通过滚动升级的方式，平滑进行升级。建议按照以
 1. 确认新版本的文件部署完成后。逐台重启 FE 和 BE 实例即可。
 2. 建议逐台重启 BE 后，再逐台重启 FE。因为通常 Doris 保证 FE 到 BE 的向后兼容性，即老版本的 FE 可以访问新版本的 BE。但可能不支持老版本的 BE 访问新版本的 FE。
 3. 建议确认前一个实例启动成功后，在重启下一个实例。实例启动成功的标识，请参阅安装部署文档。
+
+## 关于版本回滚
+因为数据库是一个有状态的服务，所以在大多数情况下，Palo/Doris 无法支持版本回滚（版本降级）。在某些情况下，可以支持 3 位或 4 位版本的回滚，但不会支持 2 位版本的回滚。
```
Comment (hf200012) [inline docs/zh-CN/installing/upgrade.md:85]: Once Doris FE is upgraded and started, it does not support version rollback, regardless of the two-digit version number or the three-digit version number

Have you tried this?

File: docs/zh-CN/installing/upgrade.md [hunk 2]
```diff
@@ -80,3 +80,12 @@ Doris 可以通过滚动升级的方式，平滑进行升级。建议按照以
...
+
+所以建议通过先升级部分节点并观察业务运行情况的方式（灰度升级）来降低升级风险。
+
+如有回滚需求，请事先联系 Palo/Doris 开发人员进行确认。
```
Comment (morningman) [inline docs/zh-CN/installing/upgrade.md:89]:
```suggestion
如有回滚需求，建议咨询 Doris 开发或运维人员。
```

File: docs/zh-CN/installing/upgrade.md (hunk 1 above)
Comment (morningman) [reply docs/zh-CN/installing/upgrade.md:85]: In many cases it is possible, but there is currently no clear guarantee. So this is a suggestion

**Review by morningman — COMMENTED** [review]
You also need to update the english document.
`docs/en/getting-started/data-partition.md`
`docs/en/installing/upgrade.md`

File: docs/zh-CN/getting-started/data-partition.md [hunk 3]
```diff
@@ -146,7 +146,7 @@ Doris 支持两层的数据划分。第一层是 Partition，支持 Range 和 Li
 
 1. Partition
 
-    * Partition 列可以指定一列或多列。分区类必须为 KEY 列。多列分区的使用方式在后面 **多列分区** 小结介绍。
+    * Partition 列可以指定一列或多列。分区列必须为 KEY 列。多列分区的使用方式在后面 **多列分区** 小结介绍。
```
Comment (EmmyMiao87) [inline docs/zh-CN/getting-started/data-partition.md:149]: Please update EN document  together ~

**Pull request closed without merging**